components/renderer/CaseStudy.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import LinkWrapper from '@components/LinkWrapper';
+import Image from 'next/image';
+import classNames from 'classnames';
+import { TypePageCaseStudy, TypePageFields } from '@types';
+import * as Contentful from 'contentful';
+
+type CaseStudyPropTypes = {
+    pageCaseStudy: Contentful.Entry<TypePageFields>;
+};
+
+const CaseStudy = ({ pageCaseStudy }: CaseStudyPropTypes) => {
+    const { title } = pageCaseStudy.fields;
+    const { fields: caseStudyPageFields } = pageCaseStudy.fields.content as TypePageCaseStudy;
+    const { summary, heroImage } = caseStudyPageFields.caseStudy.fields;
+    return (
+        <div
+            key={pageCaseStudy.fields.content.sys.id}
+            className="items-center block grid-cols-1 gap-0 mx-6 md:mx-0 md:grid md:grid-cols-14 pb-24"
```

Comment:
The `block` class isn't necessary. Also it needs spacing between the columns so the image and text don't run together. You could add `md:gap-8`